reference/SUPPORTED_QUERIES.md

```diff
@@ -0,0 +1,159 @@
+# EQL Supported Queries
+
+`eqlpy` supports querying on encrypted data through EQL's [specialized functions](https://github.com/cipherstash/encrypt-query-language#searching-data-with-eql).
+
+## Table of contents
+
+- [Query types](#query-types)
+- [Required index types](#required-index-types)
+- [Supported query interfaces](#supported-query-interfaces)
+  - [Django ORM](#django-orm)
+  - [SQLAlchemy](#sqlalchemy)
+  - [psycopg](#psycopg)
+
+## Query types
+
+The `eqlpy` library enables the following types of queries on encrypted data:
+
+- Equality where values are exactly matched, like `X = Y`
+- Partial matching where a string is a substring of another, similar to `X like Y`
+- Comparison where a value is greater than, less than, or equal to another, like `X > Y`
+  - Ordering is supported where comparison is supported, like `ORDER BY C`
+- JSONB containment where a document is contained in another, like `X @> Y`
+- JSONB value extraction where a value is extracted from a document with a path, like `$.X.Y`
+
+## Required index types
+
+- Equality queries require either the `unique` or `ore` index type
+- Partial matching of strings requires the `match` index type
+- Comparison requires the `ore` index type
+- JSONB containment requires the `ste_vec` index type
+
+TODO: add details for ejson_path
```

Comment:
Actually, I'm not sure what indexes are needed or if any index is needed here...

Reply:
[This section](https://github.com/cipherstash/encrypt-query-language/blob/main/docs/reference/JSON.md#configuring-the-index) of the EQL docs might be what you're looking for @yujiyokoo?

Reply:
What @coderdan said and EJSON paths use `ste_vec` indexes (same as containment and the other JSONB ops).

Reply:
Thank you both 🙇